impl/src/main/java/com/sevenbridges/apiclient/impl/project/DefaultMember.java

```diff
@@ -22,18 +22,22 @@
 import com.sevenbridges.apiclient.impl.resource.SubResourceProperty;
 import com.sevenbridges.apiclient.lang.Assert;
 import com.sevenbridges.apiclient.project.Member;
+import com.sevenbridges.apiclient.project.MemberType;
 import com.sevenbridges.apiclient.project.Members;
 
 import java.util.Map;
 
 public class DefaultMember extends AbstractInstanceResource implements Member {
 
   // SIMPLE PROPERTIES:
+  static final StringProperty ID = new StringProperty("id");
   static final StringProperty USERNAME = new StringProperty("username");
+  static final StringProperty EMAIL = new StringProperty("email");
+  static final StringProperty TYPE = new StringProperty("type");
   static final SubResourceProperty PERMISSIONS = new SubResourceProperty("permissions");
 
   private static final Map<String, Property> PROPERTY_DESCRIPTORS = createPropertyDescriptorMap(
-      USERNAME, PERMISSIONS
+      USERNAME, EMAIL, PERMISSIONS
```

Comment:
All new fields should be added to `PROPERTY_DESCRIPTORS` map (only `EMAIL` was added).